source/geod24/LocalRest.d

```diff
@@ -365,6 +380,16 @@ public final class RemoteAPI (API) : API
                 throw e;
     }
 
+    /// Clear up `max` messages from the pending messages
+    private static size_t removeMessages (size_t count)
```

Comment:
Oh btw, I assume you're going to use the return value of this in a future PR?

Reply:
Yeah it's something I'm considering. I will add the ability to drop X messages soon ™️